fairroot/geobase/FairGeoSet.h

```diff
@@ -76,6 +77,7 @@ class FairGeoSet : public TNamed
     const char* getGeomFile() { return geoFile.Data(); }
     Int_t getMaxSectors(void) { return maxSectors; }
     Int_t getMaxModules(void) { return maxModules; }
+    void setMaxModules(Int_t _maxMods) { maxModules = _maxMods; }
```

Comment:
I think, the general naming convention was about CamelCase, right?

Please let's at least use it for new APIs?

```suggestion
    void SetMaxModules(Int_t _maxMods) { maxModules = _maxMods; }
```

Reply:
Correct. I kept the old convention for the geobase area since this was inherited from Hades. I agree that we should change this but I would like to do it in a second step. There are many more issues with this class which I would like to fix.
The main intention of this PR was to get rid of the sprintf usage. I would like  to limit the scope of this PR to this.

Reply:
Okay, I "unresolve" this for another point:

Some of the inner workings of FairGeoSet (especially the usage of the `modules` member variable) depend on `maxModules` not changing during operation. Providing a simple setter like this suggests that changing maxModules at any time is allowed.

Either this method should check that `modules` is not already allocated.

Or (preferred) this method should not exist at all and the maxmodules parameter should be part of the ctor.

If you don't want to go the ctor route in this PR, then maybe leave out the complete maxModules part at all and move it to another PR?

Reply:
BTW: The same is probably true for `maxSectors`?